### ISSUE TITLE
TypeError: INSTRUCTOR._load_sbert_model() got an unexpected keyword argument 'token'

### ISSUE THREAD

**Issue by jwmatthews** [konveyor-ecosystem/kai#60]
```
$ make run-server                  
python ./kai/server.py
/Users/jmatthews/git/jwmatthews/kai/env/lib/python3.12/site-packages/langchain/chat_models/__init__.py:31: LangChainDeprecationWarning: Importing chat models from langchain is deprecated. Importing from langchain will no longer be supported as of langchain==0.2.0. Please import from langchain-community instead:

`from langchain_community.chat_models import ChatOpenAI`.

To install langchain-community run `pip install -U langchain-community`.
  warnings.warn(
modules.json: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████| 461/461 [00:00<00:00, 1.30MB/s]
Traceback (most recent call last):
  File "/Users/jmatthews/git/jwmatthews/kai/./kai/server.py", line 378, in <module>
    emb_provider=EmbeddingInstructor(model="hkunlp/instructor-base"),
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jmatthews/git/jwmatthews/kai/kai/incident_store_advanced.py", line 141, in __init__
    self.instructor = INSTRUCTOR(model)
                      ^^^^^^^^^^^^^^^^^
  File "/Users/jmatthews/git/jwmatthews/kai/env/lib/python3.12/site-packages/sentence_transformers/SentenceTransformer.py", line 190, in __init__
    modules = self._load_sbert_model(
              ^^^^^^^^^^^^^^^^^^^^^^^
TypeError: INSTRUCTOR._load_sbert_model() got an unexpected keyword argument 'token'
make: *** [run-server] Error 1

```

**Comment by JonahSussman** [comment]
Try https://github.com/konveyor-ecosystem/kai/pull/62

**Comment by JonahSussman** [comment]
It's related to the fact the `sentence-transformers` library had a breaking change from `2.2.2` to `2.3.0` [as shown here](https://github.com/PromtEngineer/localGPT/issues/722)

**Comment by JonahSussman** [comment]
@jwmatthews I believe that this issue should be fixed with d04e4cba5c8c47f71f047e7c89cf02d215108e3c
